Makefile

```diff
@@ -22,6 +22,21 @@ NVCC_CUDNN =
 # By default we don't build with cudnn because it blows up compile time from a few seconds to ~minute
 USE_CUDNN ?= 0
 
+# We will place .o files in the `build` directory (create it if it doesn't exist)
```

Comment:
Based on the below - please change this comment to remove (create it if it doesn't exist) - I don't think we need an explicit mkdir here.